package.json

```diff
@@ -18,6 +18,10 @@
       "email": "scottspork@gmail.com"
     }
   ],
+  "scripts": {
+    "lint": "bash ./bash/lint.sh",
+    "test": "bash ./bash/test.sh"
```

Comment:
you should be able to run `npm test` directly now

also, you may want to change the tests and replace `'##'` placeholders, for userId, with something like `process.env.USPS_USER_ID`, and then run tests via `USPS_USER_ID=## npm test`

Reply:
though, note that for lints you'd have to do `npm run lint`